src/share/poudriere/bulk.sh

```diff
@@ -54,6 +54,10 @@ Options:
     -j name     -- Run only on the given jail
     -k          -- When doing testing with -t, don't consider failures as
                    fatal; don't skip dependent ports on findings.
+    -m          -- minimal repository, only create a repository with the listed
+                   packages, incompatible with -a.
+    -M          -- medium repository, only create a repository with the listed
+                   packages and their runtime dependencies, incompatible with -a.
```

Comment:
My default reaction to new flags in PRs is to suggest or ask if this could be a hook instead. For this particular feature it seems like it could be entirely a hook in `pkgclean`. We could even provide this as an example hook.